cpu/sam0_common/periph/i2c.c

```diff
@@ -171,6 +185,17 @@ void i2c_release(i2c_t dev)
     mutex_unlock(&locks[dev]);
 }
 
+void i2c_deinit(i2c_t dev)
```

Comment:
And here `#ifdef MODULE_PERIPH_I2C_RECONFIGURE`

Reply:
Ok.